hs_core/views/__init__.py

```diff
@@ -557,6 +587,9 @@ def delete_file(request, shortkey, f, *args, **kwargs):
     res, _, user = authorize(request, shortkey, needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE)
     hydroshare.delete_resource_file(shortkey, f, user)  # calls resource_modified
     request.session['resource-mode'] = 'edit'
+
+    web_services.update(shortkey)
```

Comment:
Same comment as above: Need to put web_services.update() call into a celery task so it runs in a background thread so as not to block the main thread, especially given the calling functions are called very frequently.